Add 'More Color Tables' option to ColorTableButton

biagas opened the pull request:
### Description

It will in turn call the OpenColorTable slot in the gui.

Resolves #17797

Updated section of Pseudocolor Plot docs with new color table button image and modified text to indicate usage of the new option
.
The docs update also includes mods to one-sentance-per-line, per our format specs.


### Type of change

<!-- Please check one of the boxes below -->

* ~~[ ] Bug fix~~
* [X] New feature
* ~~[ ] Documentation update~~
* ~~[ ] Other~~ <!-- please explain with a note below -->

### How Has This Been Tested?

Tested on Windows and Linux, Clicking a color table button shows the new option, and clicking it opens the Color Table window.

### Checklist:

- [X] I have commented my code where applicable.
- [X] I have updated the release notes.
- [ ] I have made corresponding changes to the documentation.~~
- ~~[ ] I have added debugging support to my changes.~~
- ~~[ ] I have added tests that prove my fix is effective or that my feature works.~~
- ~~[ ] I have confirmed new and existing unit tests pass locally with my changes.~~
- ~~[ ] I have added new baselines for any new tests to the repo.~~
- [X] I have NOT made any changes to [*protocol* or *public interfaces*][3] in an RC branch.

[1]: https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/StyleGuide.html
[2]: https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/pr_create.html#reviewers
[3]: https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/RCDevelopment.html#communication-protocols-and-public-apis

biagas committed:
Add 'More Color Tables' option to ColorTableButton

It will in turn call the OpenColorTable slot in the gui.

biagas committed:
Fix Typo.

biagas commented:
@JustinPrivitera When you have time can you take this for a spin, see if this is what your were thinking.

biagas committed:
Merge branch 'develop' into task/biagas/connect_colortablebutton_colortablewindow

JustinPrivitera commented:
That's awesome!

biagas committed:
Add '...' to end of 'More Color Tables' label.

biagas commented:
@JustinPrivitera should this go on 3.4RC ?

JustinPrivitera commented:
> @JustinPrivitera should this go on 3.4RC ?

@biagas I think yes. We've had enough folks bring up missing color tables that I think having this in our next patch release would go a long way towards stopping the bleeding.

biagas committed:
Update release notes.

biagas commented:
Do you think this image and docs should be updated? [Changing the color table](https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/using_visit/Plots/PlotTypes/PseudocolorPlot.html#changing-the-color-table)

markcmiller86 commented:
It does not look like any state objects are changed so its probably ok on the RC.

JustinPrivitera commented:
> Do you think this image and docs should be updated? [Changing the color table](https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/using_visit/Plots/PlotTypes/PseudocolorPlot.html#changing-the-color-table)

I think yes. I can do that if you'd like. Sorry this keeps expanding into more work!

markcmiller86 commented:
> I think yes. I can do that if you'd like. Sorry this keeps expanding into more work!

FWIW...I was thinking just an update of that screen capture to include the new *top line* option and an additional sentence about what that options means and maybe why its useful (e.g. default selection may not be appropriate) would be sufficient.

markcmiller86 commented:
@JustinPrivitera...so....what does the :rocket: emoji mean?

JustinPrivitera commented:
> > I think yes. I can do that if you'd like. Sorry this keeps expanding into more work!
> 
> FWIW...I was thinking just an update of that screen capture to include the new _top line_ option and an additional sentence about what that options means and maybe why its useful (e.g. default selection may not be appropriate) would be sufficient.

I agree with @markcmiller86 

> @JustinPrivitera...so....what does the 🚀 emoji mean?

Blasting off? Hooray? Innovation? I don't know but it's more fun than thumbs up so I try to throw it in sometimes

biagas committed:
Add new colortablebutton image.

Update description of usage to include  `More Color Tables ...`.
Modified to one-sentence-per-line.

biagas committed:
Merge branch 'develop' into task/biagas/connect_colortablebutton_colortablewindow

biagas committed:
Merge branch 'develop' into task/biagas/connect_colortablebutton_colortablewindow

biagas committed:
Merge branch 'develop' into task/biagas/connect_colortablebutton_colortablewindow

JustinPrivitera commented on src/resources/help/en_US/relnotes3.4.2.html:
```diff
@@ -31,7 +31,7 @@
 <ul>
   <li>Qt6 is now the default version of Qt that is built with VisIt.</li>
-  <li>Enhancement 2</li>
+  <li>The color table button (available in plot/operator windows), has a new option 'More Color Tables ...' which will open the Color Table Windows to easier find more options.</li>
```
```suggestion
  <li>The color table button (available in plot/operator windows), has a new option 'More Color Tables ...' which will open the Color Table Window to easily find more options.</li>
```

JustinPrivitera reviewed (dismissed):
Thanks, this looks great!

biagas committed:
Update src/resources/help/en_US/relnotes3.4.2.html

Co-authored-by: Justin Privitera <35237779+JustinPrivitera@users.noreply.github.com>

biagas committed:
Merge branch 'develop' into task/biagas/connect_colortablebutton_colortablewindow

JustinPrivitera reviewed (approved):
Thanks @biagas!

biagas committed:
Merge branch 'develop' into task/biagas/connect_colortablebutton_colortablewindow
